_sass/layout/_slabs.scss

```diff
@@ -24,7 +24,10 @@
 }
 
 .slab-beta {
-  background-color: $light-gray;
+  background-color: #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.